src/containers/UserSettings/i18n/en.json

```diff
@@ -37,6 +37,9 @@
   "settings.usePaginatedTables.title": "Use paginated tables",
   "settings.usePaginatedTables.description": " Use table with data load on scroll for Nodes and Storage tabs. It will increase performance, but could work unstable",
 
+  "settings.useHideDomainDatabase.title": "Use hide domain database",
+  "settings.useHideDomainDatabase.description": "Hide root domain on database list",
```

Comment:
root = domain, it's better only leave title like "Hide domain database". And better call it "Show domain database", since it's disabled by default

Reply:
discussed with Alexey, he approved your suggestion